hack/local-up-volcano.sh

```diff
@@ -97,6 +98,21 @@ if [ "${INSTALL_MODE}" == "kind" ]; then
   export KUBECONFIG="$(kind get kubeconfig-path ${CLUSTER_CONTEXT})"
 fi
 
+echo "info: loading images into cluster"
+images=(
+    IMAGE_PREFIX-controller-manager:${TAG}
+    IMAGE_PREFIX-scheduler:${TAG}
+    IMAGE_PREFIX-webhook-manager:${TAG}
+)
+for n in ${images[@]}; do
+    echo "info: loading image $n"
+    kind load docker-image --name $CLUSTER_NAME $n
+done
+
+kubectl delete deploy volcano-admission -n $NAMESPACE
+kubectl delete deploy volcano-controllers  -n $NAMESPACE
+kubectl delete deploy volcano-scheduler   -n $NAMESPACE
+
```

Comment:
Are these changes working for environments existing Volcano installed and update it?

Reply:
Yes,it work in when code change and update image.

Reply:
> Yes,it works when we change code and need to update image.

Have you tested it at local environment?

Reply:
Sure, when we have code change ,this pr will be very convenient.

Reply:
Sorry,there are still some problems with the submission before, I have optimized it and perfected it.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/2120  kind kubeconfig-path have not supported.

Reply:
l don't think it's a good idea to hard code `volcano` here, due to the reason that we may deploy volcano using helm with some other prefix.
In addition, if we set `imagepullpolicy` to `Always`, the latest build image will be pulled every time you run this script.